packages/client/screens/MyProfile/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useInterpret, useSelector } from '@xstate/react';
+import { Button, Text } from 'dripsy';
+import React, { useMemo } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import {
+    AppScreen,
+    AppScreenContainer,
+    AppScreenHeader,
+    Typo,
+} from '../../components/kit';
+import { UserProfileScreenProps } from '../../types';
+import { getFakeUserID } from '../../contexts/SocketContext';
+import { createMyProfileInformationMachine } from '../../machines/myProfileInformationMachine';
+import { MusicPlaylistEditorUsersSearchNavigator } from '../../navigation';
+
+const MyProfileScreen: React.FC<UserProfileScreenProps> = ({ navigation }) => {
+    const insets = useSafeAreaInsets();
+    const userID = getFakeUserID();
+
+    const userProfileInformationMachineConfigured = useMemo(
+        createMyProfileInformationMachine,
```

Comment:
It does the same thing but I think people would be more familiar with this syntax:

```suggestion
        () => createMyProfileInformationMachine(),
```